### DISCUSSION TITLE
Support 3d transformations rotation like rotate-y

### DISCUSSION THREAD

**Discussion by OmarShref** [tailwindlabs/tailwindcss#14189] [Ideas]
Support 3d transformations rotation like rotate-y

**Comment by franktucker907** [comment]
Hello,
Certainly! In CSS, you can use the rotateY() method to perform 3D transformations that rotate an element around its Y-axis. Here’s a simple example:
[Login Launchpad Classlink
](https://www-mypascoconnect.com)CSS

#myDiv {
  width: 100px;
  height: 100px;
  background-color: lightblue;
  transform: rotateY(150deg);
}
